packages/grafana-data/src/types/panel.ts

```diff
@@ -120,8 +120,9 @@ export interface PanelOptionsEditorItem<TValue = any, TSettings = any> extends R
   settings?: TSettings;
 }
 
-export interface PanelOptionsEditorConfig<TSettings = any, TValue = any>
-  extends Pick<PanelOptionsEditorItem<TValue, TSettings>, 'id' | 'description' | 'name' | 'settings'> {}
+export interface PanelOptionsEditorConfig<TSettings = any> extends RegistryItem {
```

Comment:
@dprokop felt this was simpler and seems to work the Pick that only picks the props from RegistryItem felt a bit fragile

Reply:
Fragile in what way? I am fine with this change tho, the only thing i don't like is these two properties will be available when using the API: 
```
  aliasIds?: string[]; // when the ID changes, we may want backwards compatibility ('current' => 'last')
  /**
   * Some extensions should not be user selectable
   *  like: 'all' and 'any' matchers;
   */
  excludeFromPicker?: boolean;```

Not that it's a big deal, but i like to expose only the things that really matter.

Reply:
then I think we should just add those properties to this type, and not extend RegistyItem